recipe/meta.yaml

```diff
@@ -30,8 +30,7 @@ requirements:
   host:
     - python
     - numpy
-    - hdf5  # [win]
-    - hdf5-static  # [not win]
+    - hdf5
```

Comment:
If this works for the build step we'll need to add it to the run step now too I think, which would be okay I think

Reply:
The run export guarantees that

https://github.com/conda-forge/hdf5-feedstock/blob/main/recipe/meta.yaml#L88